cluster/examples/kubernetes/ceph/toolbox.yaml

```diff
@@ -17,43 +17,24 @@ spec:
     spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
-      - name: rook-ceph-tools
```

Comment:
Could you confirm your steps to test the toolbox pod changes in a running cluster?

Reply:
The pod goes into `crashLoopBackOff` after removing `privileged`